src/components/base-styles/base/links/_links.scss

```diff
@@ -15,7 +15,7 @@
 a,
 %link {
   color: color(link);
-  text-decoration: none;
+  // text-decoration: none;
```

Comment:
Could you delete this instead of commenting it out?

Reply:
Done, see 690d11c